src/transformers/default-schema-transformer.ts

```diff
@@ -35,10 +36,10 @@ function handleQuestion(question: FormField, form: FormSchema) {
   try {
     sanitizeQuestion(question);
     setFieldValidators(question);
-    transformByType(question);
+     transformByType(question);
```

Comment:
Shouldn't this be called with `await`, now that we've made it an async function?